build/src/src/calls/sshPasswordIsSecure.js

```diff
@@ -0,0 +1,18 @@
+const { isSshPasswordSecure } = require("modules/sshPassword");
+
+/**
+ * Changes the SSH password of the host for the user `dappnode`
```

Comment:
This is not the SSH password, is the system wide user password, so the naming is not the best.

Reply:
Ok, renaming all functions and variables